File: Python Word Examples/ConvertApiDemo/convertToHtml.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.convert_api.ConvertApi(configuration)

file_name = "ConvertToHtml.docx"
extra = {'folder': 'input'}
dest_file_path = 'output/ConvertToHtml/'
api.convert_to_html(file_name, dest_file_path, **extra)

File: PHP Word Examples/ConvertApiDemo/convertToHtml.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\ConvertApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ConvertApi($configuration);

$name = "ConvertToHtml.docx";
$folder = "input";
$format = 'html';
$password = null;
$dest_file_path =  "output/ConvertToHtml/";
$result=$apiInstance->convertToHtml($name,$dest_file_path, $password, $folder, $storage);